pcsx2/SIO/Sio.cpp

```diff
@@ -134,3 +134,31 @@ void AutoEject::ClearAll()
 		}
 	}
 }
+
+// Decremented once per frame if nonzero, indicates how many more frames must pass before
+// memcards are considered "no longer being written to". Used as a way to detect if it is
+// unsafe to shutdown the VM due to memcard access.
+static size_t currentBusyTicks = 0;
+
+void MemcardBusy::Decrement()
+{
+	if (currentBusyTicks)
```

Comment:
Since this is being read by the GS thread, `currentBusyTicks` should be `std::atomic_uint32_t`. This function should then be:
```cpp
if (currentBusyTicks.load(std::memory_order_relaxed) == 0)
  return;

currentBusyTicks.fetch_sub(1, std::memory_order_release);
```

(I know this is painful, but thread safety..)